content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/deploy-corda-cluster.md

```diff
@@ -11,6 +11,13 @@ section_menu: corda-5-beta
 
 This page describes how to deploy Corda 5 Beta. It assumes all necessary [prerequisites](../prerequisites.html) have been installed.
 
+{{< note >}}
+PostgreSQL and Kafka must be running before a Corda cluster is deployed but the mechanism to achieve that is up to you. You can:
+* run PostgreSQL and Kafka in containers.
+* use a managed service such as RDS.
+* run both on bare-metal hardware.
```

Comment:
The bullet list sounds a bit prescriptive which is the opposite of what I was hoping for! Maybe something like:
```suggestion
PostgreSQL and Kafka must be running before a Corda cluster is deployed but the mechanism to achieve that is up to you. For example, you can:
* run PostgreSQL and Kafka on Kubernetes.
* use a managed service such as Amazon RDS for PostgreSQL, Amazon Managed Streaming for Apache Kafka, or Confluent Cloud.
```